.travis.yml

```diff
@@ -9,13 +9,14 @@ os:
   - osx
 
 osx_image: xcode7.3
-sudo: false
+sudo: true
 dist: trusty
 
 env:
   global:
     - ERT_SHOW_BACKTRACE=1
-    - LD_LIBRARY_PATH="$(pwd)/install/lib64"
+    - INSTALL_DIR="$(pwd)/install"
+    - LD_LIBRARY_PATH="$INSTALL_DIR/lib64"
```

Comment:
The library suffix - i.e. `lib64` in this case - depends on platform, it is `lib64` on 64 bit RedHat but e.g. `lib` on Ubuntu. CMake wise this is set [here](https://github.com/equinor/libecl/blob/master/cmake/Modules/UseMultiArch.cmake), so for the current travis environment I do not think the `LD_LIBRARY_PATH=$INSTALL_DIR/lib64` will be correct.

Reply:
> OS/Arch:      linux/amd64

So actually I think we're good :) But I'll add the `lib` version as well to the path.